addons/pos_sale/static/src/js/OrderManagementScreen/SaleOrderManagementScreen.js

```diff
@@ -233,6 +233,7 @@ odoo.define('pos_sale.SaleOrderManagementScreen', function (require) {
                 }
               }
 
+              currentPOSOrder.set_to_invoice(true);
```

Comment:
Why? you don't need to invoice all pos order comming from a SO. Example, the customer don't want an invoice